locale/ko/index.md

```diff
@@ -11,8 +11,8 @@ labels:
   tagline-lts: 안정적, 신뢰도 높음
   changelog: 변경사항
   api: API 문서
-  version-schedule-prompt: LTS 일정은
-  version-schedule-prompt-link-text: 여기서 확인하세요
+  version-schedule-prompt: For information about supported releases, see the
+  version-schedule-prompt-link-text: release schedule
```

Comment:
@nodejs/nodejs-ko  Can someone provide translations for these phrases (which will be put together to form a sentence in the page)?

Reply:
```suggestion
  version-schedule-prompt: 지원되는 릴리스에 관한 정보는
  version-schedule-prompt-link-text: 릴리스 일정을 참고하세요
```
Here it is. :)